yarn.lock

```diff
@@ -5518,6 +5520,11 @@ data-urls@^1.1.0:
     whatwg-mimetype "^2.2.0"
     whatwg-url "^7.0.0"
 
+date-fns-tz@^1.1.1:
+  version "1.1.1"
+  resolved "https://registry.yarnpkg.com/date-fns-tz/-/date-fns-tz-1.1.1.tgz#2e0dfcc62cc5b7b5fa7ea620f11a5e7f63a7ed75"
+  integrity sha512-5PR604TlyvpiNXtvn+PZCcCazsI8fI1am3/aimNFN8CMqHQ0KRl+6hB46y4mDbB7bk3+caEx3qHhS7Ewac/FIg==
+
```

Comment:
I'm guessing these packages are very small? Because they are being imported into a site-wide component, so all of our bundles will take that increase. I heard Date FN smaller than Moment JS but if we don't have any plans to completely replace Moment JS then currently Date FN is just increasing bundle sizes.

Reply:
I did a `yarn build:webpack:prod` on this branch as well as on `master`.

This is the result of a prod webpack build on this branch:

![image](https://user-images.githubusercontent.com/2008881/107981443-ec869880-6f76-11eb-994e-48e6064713ae.png)

Bundle sizes of 1.1M

And on master:

![image](https://user-images.githubusercontent.com/2008881/107981582-32436100-6f77-11eb-8d1c-61142b3b7090.png)

---

The difference here seems to be very small, so I broke it down a bit further:

![image](https://user-images.githubusercontent.com/2008881/107982530-104ade00-6f79-11eb-9576-1632d5a9aabe.png)


On the built `static-pages` entry file on this branch with `date-fns` included, it ended up being 553 bytes larger.